frontend/src/component/commandBar/RecentlyVisited/CommandResultGroup.tsx

```diff
@@ -193,7 +190,10 @@ export const CommandResultGroup = ({
     children,
 }: CommandResultGroupProps) => {
     const { trackEvent } = usePlausibleTracker();
-    if (!children && (!items || items.length === 0)) {
+    if (
```

Comment:
Fixed a bug, because if no children was passed it, it passed in empty array, so just `!children` was not working